src/components/Notifications/Notification.js

```diff
@@ -26,19 +26,12 @@ const Notification = (props) => {
     className,
   ];
 
-  const wrapIcon = () => {
-    const iconContainerClasses = [
-      { 'slds-icon_container': true },
-      'slds-m-right_small slds-no-flex slds-align-top',
-    ];
-
-    return (
-      <span className={cx(iconContainerClasses)}>
-        {icon}
-        <span className="slds-assistive-text">{title}</span>
-      </span>
-    );
-  };
+  const wrapIcon = () => (
+    <span className={'slds-icon_container slds-m-right_small slds-no-flex slds-align-top'}>
```

Comment:
This should be plain string prop: `className="slds-icon_container slds-m-right_small slds-no-flex slds-align-top"`